proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -78,8 +78,8 @@ Die englische Übersetzung dient lediglich der Information.
    5. den Aufbau einer inklusiven und vielfältigen Gemeinschaft, in der
       Mitwirkende aller Hintergründe, Geschlechter und Fähigkeitsstufen
       willkommen sind
-   6. die Aufrechterhaltung der Entwicklungsinfrastruktur für niedrige
-      Einstiegshürden
+   6. die Aufrechterhaltung ein Entwicklungsinfrastruktur für niedrige
+      Einstiegshürden hinsichtlich Soft- und Hardware.
```

Comment:
Why this addional limitation "hinsichtlich Soft- und Hardware". The previous wording covered all other infrastructure like Github, Transifex, GMail etc. too. Let's keep this wording open.

Reply:
It was my failed try to acknowledge Eve's idea and clarify the sharing controllers is also the Association purpose. Do you have an alternative idea to mention it in a way without limiting us by details.

Reply:
I think the old text allowed everything.

And such a controller sharing needs a good concept, that the e.V. doesn't loose the non-profit state. Therefore we must neither generate to much income, nor should we give monetary value (free controllers) to individuals. For me this is an additional complex topic, which we shouldn't define in the Satzung. With an open text, we can define this anytime later, when our Schatzmeister became familar with all these tax related topics.

Reply:
OK, I will revert to the old version.